sklearn/manifold/_barnes_hut_tsne.pyx

```diff
@@ -210,18 +204,16 @@ cdef void compute_gradient_negative(float[:,:] pos_reference,
     if qt.verbose > 20:
         printf("[t-SNE] Tree: %li clock ticks | ", dta)
         printf("Force computation: %li clock ticks\n", dtb)
-    free(iQ)
-    free(force)
-    free(pos)
+
+    # Liberate the ressources
```

Comment:
"Free the resources" would be more correct but this comment is useless anyway. Please just remove it.